src/main/java/com/dabsquared/gitlabjenkins/GitLabPushTrigger.java

```diff
@@ -115,6 +116,9 @@ public static void migrateJobs() throws IOException {
                     if (trigger.addCiMessage) {
                         project.getPublishersList().add(new GitLabCommitStatusPublisher());
                     }
+                    if (trigger.addNoteOnMergeRequest) {
```

Comment:
This migration won't be triggered for anybody who already uses version 1.2.x of the plugin as for them the property jobsMigrated of the GitLabPushTrigger is already set to true. So this needs to be placed in a separate "migration block".